python/MaterialXTest/tests_to_html.py

```diff
@@ -14,27 +14,42 @@
 
 def createDiff(image1Path, image2Path, imageDiffPath):
     try:
+        if os.path.exists(imageDiffPath):
+            os.remove(imageDiffPath)
+        
+        if not os.path.exists(image1Path):
+            print ("Image diff input missing: " + image1Path)
+            return
+        
+        if not os.path.exists(image2Path):
+            print ("Image diff input missing: " + image2Path)
+            return
+
         image1 = Image.open(image1Path).convert('RGB')
         image2 = Image.open(image2Path).convert('RGB')
         diff = ImageChops.difference(image1, image2)
         diff.save(imageDiffPath)
     except Exception:
+        if os.path.exists(imageDiffPath):
+            os.remove(imageDiffPath)
         print ("Failed to create image diff between: " + image1Path + ", " + image2Path)
 
 def main(args=None):
 
     parser = argparse.ArgumentParser()
     parser.add_argument('-i', '--inputdir', dest='inputdir', action='store', help='Input directory', default=".")
     parser.add_argument('-i2', '--inputdir2', dest='inputdir2', action='store', help='Second input directory', default=".")
+    parser.add_argument('-i3', '--inputdir3', dest='inputdir3', action='store', help='Third input directory', default=".")
     parser.add_argument('-o', '--outputfile', dest='outputfile', action='store', help='Output file name', default="tests.html")
     parser.add_argument('-d', '--diff', dest='CREATE_DIFF', action='store_true', help='Perform image diff', default=False)
     parser.add_argument('-t', '--timestamp', dest='ENABLE_TIMESTAMPS', action='store_true', help='Write image timestamps', default=False)
     parser.add_argument('-w', '--imagewidth', type=int, dest='imagewidth', action='store', help='Set image display width', default=256)
     parser.add_argument('-ht', '--imageheight', type=int, dest='imageheight', action='store', help='Set image display height', default=256)
     parser.add_argument('-cp', '--cellpadding', type=int, dest='cellpadding', action='store', help='Set table cell padding', default=0)
     parser.add_argument('-tb', '--tableborder', type=int, dest='tableborder', action='store', help='Table border width. 0 means no border', default=3)
-    parser.add_argument('-sl', '--sourcelang', dest='sourcelang', action='store', help='Source language. Default is source', default="glsl")
-    parser.add_argument('-dl', '--destlang', dest='destlang', action='store', help='Destination language. Default is dest', default="osl")
+    parser.add_argument('-sl', '--sourcelang', dest='sourcelang', action='store', help='Source language. Default is glsl', default="glsl")
+    parser.add_argument('-dl', '--destlang', dest='destlang', action='store', help='Destination language. Default is osl', default="osl")
+    parser.add_argument('-dl2', '--destlang2', dest='destlang2', action='store', help='Second destination language. Default is empty', default="")
```

Comment:
For consistency with the "inputdir" options above, perhaps we should name these three arguments "lang1", "lang2", and "lang3"?  To my mind, that seems clearer than having a single "source" language and two "destination" languages, and the slight reduction in backwards compatibility seems acceptable for a script used in render testing.